designsafe/static/scripts/ng-designsafe/components/modals/data-browser-service-move/data-browser-service-move.component.js

```diff
@@ -26,9 +26,13 @@ class DataBrowserServiceMoveCtrl {
         this.state = {
             busy: false,
             error: null,
-            listingProjects: false
+            listingProjects: false,
+            listingProjects: false,
```

Comment:
Is there a reason `listingProjects: false,` is here twice?

Reply:
Nope, I'll fix that.

Reply:
```suggestion
```